source/_integrations/enphase_envoy.markdown

```diff
@@ -685,6 +685,95 @@ The Envoy Metered in multiphase setup, sums the voltages of the phases measured
 
 In multiphase installations with batteries, in countries with phase-balancing grid meters, the battery will export to the grid on one phase the amount it lacks on another phase. This other phase pulls the missing amount from the grid, as if it is using the grid as a 'transport' between phases. Since the grid meter will balance the amount imported and exported on the two phases, the net result is zero. The Envoy multiphase net-consumption CTs, however, will report the amounts on both phases, resulting in too high export on one and too high import on the other. One may consider using the `lifetime balanced net energy consumption` which is the sum of grid import and export to eliminate this effect. This would require some templating to split these values into import and export values. Alternatively, use the `current net power consumption` or `balanced net power consumption` with a Riemann integral sum helper.
 
+## Technical Information
+
+This integration uses the [pyenphase library](https://pypi.org/project/pyenphase/) to collect all Envoy data. Depending on the version of Home Assistant you are running, this integration may be using different versions of the library.
+
+### Setup
+
+When a configuration entry for this integration is newly added, or each time a configured entry is (re)loaded, the Envoy setup is performed. Using the specified DNS name or IP address, either from the user input, or from the configuration entry, the `/info` endpoint on the Envoy is read using a GET request. This returns XML data containing, among other information, the Envoy serial-number, the hardware version and the running firmware version. No username, password or tokens are needed in this stage of the process.
```

Comment:
I am really confused why this is here? This is developer information, not end-user documentation.